packages/next/server/base-server.ts

```diff
@@ -689,11 +670,14 @@ export default abstract class Server {
       )
     }
 
-    const page: { name?: string; params?: { [key: string]: string } } = {}
+    const page: RequestData['page'] = {}
```

Comment:
I'm not 100% sold on this. This type includes a lot of data right? Like the Geolocation and so on

Reply:
It should be the same with  the original type. Do you mean `RequestData`?